src/js/components/NameValueList/__tests__/NameValueList-test.tsx

```diff
@@ -113,6 +113,24 @@ describe('NameValueList', () => {
     expect(asFragment()).toMatchSnapshot();
   });
 
+  test(`should render correct width with max and min`, () => {
+    const { asFragment } = render(
+      <Grommet>
+        <NameValueList
+          nameProps={{ width: { min: 'xsmall', max: 'max-content' } }}
```

Comment:
Do we have a test scenario which covers `valueProps.width`?

Reply:
will add one! Also will also add to grommet site

Reply:
https://github.com/grommet/grommet-site/pull/537